modules/test/src/main/java/com/devonfw/module/test/common/base/SystemTest.java

```diff
@@ -11,7 +10,7 @@
  *
  * @see CategorySystemTest
  */
-@Category(CategorySystemTest.class)
+@Tag("system")
```

Comment:
If we have all these tags redundatly, shouldnt we create a constant class/interface (just like `SpringProfileConstants`)?

Reply:
Created a meta annotation for all Tags